.github/CODEOWNERS

```diff
@@ -1,7 +1,6 @@
-# CODEOWNERS file for spv-wallet-admin
+# CODEOWNERS file for Bitcoin SV
 
```

Comment:
remove those lines like in the other repositories

Reply:
Done